### DISCUSSION TITLE
[2906030]魔物联盟自走棋 Monster United

### DISCUSSION THREAD

**Discussion by ZEERDEER** [pjy612/SteamManifestCache#621] [Share]
### Steam 游戏ID

2906030

### 密钥列表

```hash
2906031;7c7c8fe085d6c210bb417a7e159fbc49394934cf71b81eebb727933d83c79924
```

### 清单附件

[Manifest.zip](https://github.com/pjy612/SteamManifestCache/files/15130473/Manifest.zip)
[Achievement.zip](https://github.com/pjy612/SteamManifestCache/files/15130484/Achievement.zip)

**Comment by pjy612** [comment]
* 分隔符错了

**Reply by ZEERDEER** [reply]
我没错，错的是这个世界。
